src/System.Memory/Common/src/System/MutableDecimal.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should this be in corefx/src/Common/ instead of System.Memory/Common/...?

Reply:
I think the concept is a bit too icky to advertise it broadly.

Reply:
@AtsushiKan, would you be ok with me moving this file within the src directory?

So, from System.Memory/Common/src/System/MutableDecimal.cs to System.Memory/src/System/MutableDecimal.cs? It seems strange to put this in a "Commons" directory and yet it is only accessed within System.Memory. At the very least, I think we should move it inside System.Memory/src, if we want to keep it in Common. Right now, System.Memory is the only one with a folder other than pkg/ref/src/tests at the root.

Reply:
It's also weird for a test project to reach into a sibling src directory. If the current setup is really objectionable, then go ahead and move it to the global Common directory. We have other narrowly-targeted stuff up there so this isn't that different.